Realm.podspec

```diff
@@ -147,6 +147,7 @@ Pod::Spec.new do |s|
                                 'OTHER_LDFLAGS' => '$(REALM_LD_CLASSIC)',
                               }
   s.preserve_paths          = %w(include scripts)
+  s.resource_bundles        = {'realm_objc_privacy' => ['Realm/PrivacyInfo.xcprivacy']}
```

Comment:
https://github.com/SDWebImage/SDWebImage/pull/3649/files What I'm seeing in some other SDKs is they use the name of the framework as the key, so shouldn't be 
s.resource_bundles = {'Realm' => ['Realm/PrivacyInfo.xcprivacy']}, or maybe it is irrelevant given that we don't know if it is going to even work

Reply:
I based this on what [Flutter](https://github.com/flutter/packages/pull/5846) is doing since they appeared to put a bunch of effort into looking into things.

Reply:
> What I'm seeing in some other SDKs is they use the name of the framework as the key

FYI, see [this comment](https://github.com/flutter/packages/pull/5846#issuecomment-1881788602) for the relevant discussion about why we aren't doing that.